en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md

```diff
@@ -555,10 +561,10 @@ For more information, see [Download and Initialize the CTL Tool](#download-and-i
             **Flags:**  
             
             -   Required :  
-                `--environment` or `-e` : Environment of which the API state should be changed  
-                `--name` or `-n` : Name of the API to be state changed  
-                `--version` or `-v` : Version of the API to be state changed  
-                `--action` or `-a` : Action to be taken to change the status of the API
+                `--environment` or `-e` : The environment of which the API state should be changed  
+                `--name` or `-n` : The name of the API to be state changed  
+                `--version` or `-v` : The version of the API that its state needs to be changed
+                `--action` or `-a` : The action to be taken to change the status of the API
```

Comment:
Shall we change the description to make it less verbose? The current descriptions are not adding much value. I feel that we can simplify this a lot, such as,

```
                `--environment` or `-e` : The environment the command is executed on
                `--name` or `-n` : The name of the respective API
                `--version` or `-v` : The version of the respective API
                `--action` or `-a` : The action to be taken to change the status of the API

        -   Optional :  
                `--provider` or `-r` : The provider of the respective API
```

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/2559/commits/708589ec269c76201da4254ec4d018824fa1cfd8